RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -249,7 +250,8 @@
 #photonConvTrajSeedFromQuadruplets.primaryVerticesTag = cms.InputTag('pixelVertices')
 from Configuration.Eras.Modifier_trackingLowPU_cff import trackingLowPU
 trackingLowPU.toModify(photonConvTrajSeedFromSingleLeg, primaryVerticesTag   = "pixelVertices")
-
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+pp_on_AA_2018.toModify(photonConvTrajSeedFromSingleLeg.RegionFactoryPSet.RegionPSet, ptMin = 999.)
```

Comment:
sadly, this turned out to have even less effect.
I was just looking at parameters that look like a pt cut. Clearly, just having an apparently appropriate name is not good enough.

At a closer look, it seems like ```pp_on_AA_2018.toModify(photonConvTrajSeedFromSingleLeg, vtxMinDoF = 999999.)``` should actually do the job. This time I actually checked.